src/cli/project/version/bump.rs

```diff
@@ -0,0 +1,40 @@
+use crate::Project;
+use clap::Parser;
+
+pub enum BumpType {
+    Major,
+    Minor,
+    Patch,
+}
+
+/// Bump the project version.
+#[derive(Parser, Debug, Default)]
+pub struct Args {}
+
+pub async fn execute(bump_type: BumpType, mut project: Project, _args: Args) -> miette::Result<()> {
+    let current_version = project.version().as_ref().unwrap().clone();
+
+    // NOTE(hadim): logic is the same everywhere (bump last component)
+    // The correct logic should be added on rattler probably.
+    let new_version = match bump_type {
+        BumpType::Major => current_version.bump(),
+        BumpType::Minor => current_version.bump(),
+        BumpType::Patch => current_version.bump(),
+    };
```

Comment:
Should I work on a PR on the rattler side to provide `version.bump_{minor|major|patch}()` or maybe `version.bump(BumpType::{Major|Minor|Patch|Last})`?

Reply:
I think that would be awesome! With the bump type is nice.

Reply:
That said maybe remove it for now from this PR so we don't have to hold it back untill its in rattler.

Reply:
Sounds good. See https://github.com/prefix-dev/pixi/issues/581 and https://github.com/mamba-org/rattler/issues/449

Reply:
I removed the command for now.